testSrc/unit/io/flutter/utils/AsyncRateLimiterTest.java

```diff
@@ -111,11 +108,13 @@ public void rateLimited() {
     checkLog("EVENT", "EVENT", "EVENT", "EVENT", "DONE");
 
     // First event should occur immediately so don't count it.
-    final double requestsPerSecond = (expectedEvents - 1) / (delta * 0.001);
+    // The last event just fired so add in MS_PER_EVENT to factor in that it
+    // will be that long before the next event fires.
+    final double requestsPerSecond = (numEvents - 1) / ((delta + MS_PER_EVENT) * 0.001);
 
     // Use an epsilon of 10% for comparisons to TEST_FRAMES_PER_SECOND.
```

Comment:
Is this comment still valid?

Reply:
Good point. Fixed the bug in the test that required it. Removed.